api-spec/api-spec.md

```diff
@@ -38,28 +40,40 @@ STAC API.
 
 ## HTTP Request Methods and Content Types
 
-The OAFeat and STAC APIs follow a RESTful model. A core principal of this is the use of HTTP Request Methods (verbs) and
-the `Content-Type` header to drive behavior. This section describes how these are used in the OAFeat and STAC endpoints. 
+The OAFeat and STAC APIs follow a RESTful model. A core principal of this is the use of HTTP Request Methods ("verbs") and
+the `Content-Type` header to drive behavior on resources ("nouns"). This section describes how these are used in the OAFeat and STAC endpoints. 
 
-1. **Required** GET (both OAFeat and STAC)
-2. **Recommended** POST `Content-Type: application/x-www-form-urlencoded` with the corresponding content body format.
-3. **Recommended** POST `Content-Type: multipart/form-data` with the corresponding content body format.
-4. **Optional** **STAC endpoint /search only** POST `Content-Type: application/json`, where the content body is a JSON 
-object representing a filter, as defined in the [STAC API OpenAPI specification document](STAC.yaml).  
-5. **Prohibited** **OAFeat endpoints only** POST `Content-Type: application/json`, where the content body is a JSON 
-object representing a filter. This is prohibited due to conflict with the 
+#### GET
+
+**Required**: OAFeat 1.0 only specifies GET query parameters, and they are all required. STAC's cross-collection `/search` also requires
```

Comment:
Are we talking about endpoints or parameters here? I think originally it was about endpoints.